### ISSUE TITLE
No kinetic support for recon

### ISSUE THREAD

**Issue by pauldardeau** [swiftstack/kinetic-swift#56]
Recon support for kinetic is needed for cluster monitoring

**Comment by dudymas** [comment]
Hey, are there any low-hanging fruit on this? Looking at some of the commands, maybe this could be a 'softened' to just a few different recon stats at a time?

**Comment by clayg** [comment]
Yes, I think improving the Seagate monitoring tools is more important that adapting the swift specific tools to kinetic devices.

This is a low priority enhancement.
